ci/verify_bundle.py

```diff
@@ -46,73 +88,84 @@ def verify_metadata_format(bundle_path: str):
     )
 
 
-def verify_data_shape(inference_file: str, bundle_path: str):
+def verify_data_shape(bundle_path: str, net_id: str, config_file: str):
     """
     This function is used to verify the data shape of network.
 
     """
     verify_net_in_out(
-        net_id="network_def",
+        net_id=net_id,
         meta_file=os.path.join(bundle_path, "configs/metadata.json"),
-        config_file=os.path.join(bundle_path, f"configs/{inference_file}"),
+        config_file=os.path.join(bundle_path, config_file),
         bundle_root=bundle_path,
     )
 
 
-def verify_export_torchscript(inference_file: str, bundle_path: str):
+def verify_export_torchscript(bundle_path: str, net_id: str, config_file: str):
     """
     This function is used to verify if the checkpoint is able to torchscript.
 
     """
     ckpt_export(
-        net_id="network_def",
+        net_id=net_id,
         filepath=os.path.join(bundle_path, "models/verify_model.ts"),
         ckpt_file=os.path.join(bundle_path, "models/model.pt"),
         meta_file=os.path.join(bundle_path, "configs/metadata.json"),
-        config_file=os.path.join(bundle_path, f"configs/{inference_file}"),
+        config_file=os.path.join(bundle_path, config_file),
         bundle_root=bundle_path,
     )
 
 
+def get_net_id_config_name(bundle_name: str):
+    """
+    Return values of arguments net_id and config_file.
+    """
+    net_id = "network_def"
+    config_file = "configs/inference.json"
+    if bundle_name not in custom_net_config_dict.keys():
+        return net_id, config_file
+
+    name_dict = custom_net_config_dict[bundle_name]
+    if "net_id" in name_dict.keys():
+        net_id = name_dict["net_id"]
+    if "config_file" in name_dict.keys():
+        config_file = name_dict["config_file"]
+
+    return net_id, config_file
```

Comment:
I would suggest to simplify this function:
```py
def get_net_id_config_name(bundle_name: str):
    name_dict = custom_net_config_dict.get(bundle_name, {})
    return name_dict.get("net_id", "network_def"), name_dict.get("config_file", "configs/inference.json")
```
And maybe also define these hard-code strings in the `bundle_custom_data` python file?
"net_id", "network_def", "config_file", "configs/inference.json"

Thanks.

Reply:
Thanks @Nic-Ma ! This suggestion makes the function more clear